src/main/kotlin/minesweeper/domain/Board.kt

```diff
@@ -0,0 +1,23 @@
+package minesweeper.domain
+
+class Board private constructor(
+    private val size: BoardSize,
+    private val mines: Mines,
+) {
+    val boardSize: BoardSize
+        get() = size
+    val minePositions: Mines
+        get() = mines
```

Comment:
```suggestion
class Board private constructor(
    val boardSize: BoardSize,
    val minePositions: Mines,
) {
```

정호님은 이 두 코드가 어떤 차이가 있다고 생각하셨을까요?

Reply:
get()으로 쓴게 내부 데이터를 숨기고, 외부에는 게터를 통해서만 노출하니 내부 구현이 변경가능한 상황에서 조금 더 유연하다고 생각합니다!
근데 사실 별개로 저건 코틀린 get()에 좀 익숙해지고 싶어서 그냥 생각나면 사용했습니다😂